docs/nzxt-smart-device-v2.md

```diff
@@ -0,0 +1,102 @@
+
+Additionally, it features two independent lighting (Addressable RGB) channels, unlike the single channel in the original. NZXT Aer RGB 2 fans and HUE 2 lighting accessories (HUE 2 LED strip, HUE 2 Unerglow, HUE 2 Cable Comb) can be connected. The firmware installed on the device exposes several color presets, most of them common to other NZXT products.
+
+HUE 2 and HUE+ devices (including Aer RGB and Aer RGB 2 fans) are supported, but HUE 2 components cannot be mixed with HUE+ components in the same channel. Each lighting channel supports up to 6 accessories and a total of 40 LEDs.
+
+A microphone is still present onboard for noise level optimization through CAM and AI.
+
+All configuration is done through USB, and persists as long as the device still gets power, even if the system has gone to Soft Off (S5) state.  The device also reports the state of each fan channel, as well as speed and duty (from 0% to 100%).
+
+Most capabilities available at the hardware level are supported, but other features offered by CAM, like noise level optimization and presets based on CPU/GPU temperatures, have not been implemented.
+
+
+## Initialization
+
+After powering on from Mechanical Off, or if there have been hardware changes, the device must first be initialized. Only then monitoring, proper fan control and all lighting effects will be available.
+
+```
+# liquidctl initialize
+```
+
+
+## Monitoring
+
+The device can report fan information for each channel, the noise level at the onboard sensor, as well as the type of the connected LED accessories.
+
+```
+# liquidctl status
+Device 1, NZXT Smart Device V2 (Experimental)
+Fan 2 duty                         40  %
+Fan 2 speed                       873  rpm
+Firmware version                1.5.0  
+LED 1 accessory 1     HUE 2 LED Strip  
+LED 1 accessory 2     HUE 2 Underglow  
+LED 1 accessory 3     HUE 2 Underglow  
+LED 2 accessory 1    AER RGB 2 140 mm  
+LED 2 accessory 2    AER RGB 2 140 mm  
+LED 2 accessory 3    AER RGB 2 140 mm  
+LED 2 accessory 4    AER RGB 2 120 mm  
+Noise level                        64  dB
+```
+
+
+## Fan speeds
+
+Fan speeds can only be set to fixed duty values.
+
+```
+# liquidctl set fan2 speed 90
+```
+
+| Channel | Minimum duty | Maximum duty | Note |
+| --- | --- | --- | - |
+| fan1 | 0% | 100% ||
+| fan2 | 0% | 100% ||
+| fan3 | 0% | 100% ||
+| sync | 0% | 100% | all available channels |
+
+*Always check that the settings are appropriate for the use case, and that they correctly apply and persist.*
+
+
+## RGB lighting
+
+The device features two lighting channels, `led1` and `led2`, that support up to 6 lighting accessories each. Compatible Aer RGB 2 and Aer RGB fans must be on their own channel, separate from other lighting accessories. Color modes can be set independently for each lighting channel. The specified color mode will then apply to all devices daisy chained on that channel.
```

Comment:
As far as I knew there was no restriction on mixing Aer RGB 2 fans and HUE 2 accessories.

Reply:
This might be from an earlier version of the file. The last-committed version has a re-written comment section at the top of the file where it's stated only that HUE+ and HUE 2 accessories cannot be combined on the same channel.

Reply:
Right.  In this case, this can become:

```
The device features two lighting channels, `led1` and `led2`, that support up to 6 lighting accessories each.  Color modes can be set independently for each lighting channel, but the specified color mode will then apply to all devices daisy chained on that channel.
```

Reply:
Note: I just edited the suggested change in the previous comment.

Reply:
Change made. Pending next commit.